packages/scheduler/src/components/FullCalendar/FullCalendar.js

```diff
@@ -83,92 +76,75 @@ const FullCalendar = ({
+    onClick={onCalendarRemoveAll}
+  />
 
   const changeRange = () => {
-    const view = calendarEl.current.calendar.view
+    const {view} = calendarEl.current.getApi()
     if (onDateRangeChange) {
       onDateRangeChange({startDate: view.activeStart, endDate: view.activeEnd})
     }
   }
 
   const changeView = (view = 'timelineDay') => {
-    calendarEl.current.calendar.changeView(view)
+    calendarEl.current.getApi().changeView(view)
     changeRange()
   }
 
-  const resourceRender = renderInfo => {
-    const element = renderInfo.el.getElementsByClassName('fc-cell-content')[0]
-    const checkbox = document.createElement('INPUT')
-    checkbox.type = 'checkbox'
-    checkbox.checked = 'checked'
-    checkbox.className = 'remove-resource-checkbox'
-    checkbox.onclick = () => {
-      onCalendarRemove(renderInfo.resource.extendedProps.entityKey, renderInfo.resource.extendedProps.entityModel)
-    }
-    element.prepend(checkbox)
-  }
-
-  const eventRender = ({event, el}) => {
-    const time = moment(event.start).twix(moment(event.end)).format({monthFormat: 'MMMM', dayFormat: 'Do'})
+  function renderEventContent(eventInfo) {
+    const time = moment(eventInfo.event.start)
+      .twix(moment(eventInfo.event.end))
+      .format({monthFormat: 'MMMM', dayFormat: 'Do'})
     const tooltipDescriptionContent = <div>
-      <FormattedValue type="html" value={event.extendedProps.description}/>
+      <FormattedValue type="html" value={eventInfo.event.extendedProps.description}/>
       <p>{time}</p>
-      <p><Conflict conflictStatus={event.extendedProps.conflict} intl={intl}/></p>
+      <p><Conflict conflictStatus={eventInfo.event.extendedProps.conflict} intl={intl}/></p>
     </div>
 
-    const content = ReactDOMServer.renderToString(tooltipDescriptionContent)
-
-    // eslint-disable-next-line no-new
-    new Tooltip(el, {
-      title: content,
-      placement: 'bottom',
-      trigger: 'hover',
-      container: wrapperEl.current,
-      html: true,
-      boundariesElement: 'body',
-      innerSelector: '.na-tooltip-inner',
-      arrowSelector: '.na-tooltip-arrow',
-      template: `<div class="na-tooltip" role="tooltip"><div class="na-tooltip-arrow">
-                    </div><div class="na-tooltip-inner"></div></div>
-                   `
-    })
+    return (
+      <Popover
+        content={tooltipDescriptionContent}
+        isPlainHtml
+        placement="top"
+        spacer={10}
+      >
+        <div className={`fc-event-main-frame ${eventInfo.event.extendedProps.styleAttr} `}>
+          {eventInfo.timeText && <div className="fc-event-time">{ eventInfo.timeText }</div>}
+          <div className="fc-event-title-container">
+            <div className="fc-event-title fc-sticky">
+              {eventInfo.event.title || <>&nbsp;</>}
+            </div>
+          </div>
+        </div>
+      </Popover>
+    )
   }
 
-  const FullCalendarMemorized = useMemo(() =>
+  const memoizedFullcalendar = useMemo(() =>
```

Comment:
typo? memo**r**izedFullcalendar

Reply:
Nope. Yilmaz pointed that out to me:
https://reactjs.org/docs/hooks-reference.html#usememo